Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -123,8 +124,8 @@ lemma fin3_def (P : Fin 3 → R) : ![P x, P y, P z] = P := by
 lemma fin3_def_ext (X Y Z : R) : ![X, Y, Z] x = X ∧ ![X, Y, Z] y = Y ∧ ![X, Y, Z] z = Z :=
   ⟨rfl, rfl, rfl⟩
 
-lemma comp_fin3 {S} (f : R → S) (X Y Z : R) : f ∘ ![X, Y, Z] = ![f X, f Y, f Z] := by
-  ext i; fin_cases i <;> rfl
+lemma comp_fin3 {S} (f : R → S) (X Y Z : R) : f ∘ ![X, Y, Z] = ![f X, f Y, f Z] :=
+  (FinVec.map_eq _ _).symm
```

Comment:
Untested:
```suggestion
  (FinVec.map_eq ..).symm
```

Reply:
Since we already have `FinVec.map_eq`, do we still need `comp_fin3`? Otherwise, should this be `comp_fin3_ext`, and does it make sense to have an analogous `comp_fin3` for `P : Fin 3 -> R`?

Reply:
As I [commented](https://github.com/leanprover-community/mathlib4/pull/13846#discussion_r1643750769), it can't be used to rewrite to the desired form directly.